src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -61,21 +69,48 @@ public Integer getNbConges() {
     public Integer getNbRtt(){
         return getNbRtt(LocalDate.now());
     }
-
-    public Integer getNbRtt(LocalDate d){
-        int i1 = d.isLeapYear() ? 365 : 366;int var = 104;
-        switch (LocalDate.of(d.getYear(),1,1).getDayOfWeek()){
-        case THURSDAY: if(d.isLeapYear()) var =  var + 1; break;
+/*
+ Méthode permettant de calculer le nb de jour de RTT dans l'année selon la formule
+     * nb jours rtt
+     * nombre de jour travaillés dans l'année en plein temps
+     * nombre de samedi et dimanche dans l'année
+     * nombre de jours feries ne tombant pas le week-end
+     * nombre de conges payes
+
+     @param dateReference la date a laquele on va calculer le nb de rtt pour l'annee
+     @return le nombre de jours de RTT pour l'employel'annee de la date de reference
+ */
+public Integer getNbRtt(LocalDate d ) {
+    //System.out.println(d);
+    int nbJoursAnnee = d.isLeapYear() ? 366 : 365;
+    int nbSamediDimache = 104;
+    switch (LocalDate.of(d.getYear(), 1, 1).getDayOfWeek()) {
+        case THURSDAY:
+            if (d.isLeapYear()) nbSamediDimache = nbSamediDimache + 1;
```

Comment:
Eviter d'utiliser des if sans {} pour la lisibilité entre autres
```suggestion
            if (d.isLeapYear()) {
                nbSamediDimache = nbSamediDimache + 1;
            }
```